src/main/java/mate/academy/booksstore/model/User.java

```diff
@@ -14,6 +14,7 @@
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
+import lombok.experimental.Accessors;
```

Comment:
1 Using lombok.experimental.* is not really desirable. 
2 Changing your code to test it is bad testing. 

```suggestion
import lombok.experimental.Accessors;
```